web/app/themes/justicejobs/src/js/modal.js

```diff
@@ -164,3 +148,40 @@ function enableBodyScrolling() {
   document.body.style.top = '';
   window.scrollTo(0, parseInt(scrollY || '0') * -1);
 }
+
+
+// Based on example by Hidde de Vries
+// https://hiddedevries.nl/en/blog/2017-01-29-using-javascript-to-trap-focus-in-an-element
+
+function trapFocus() {
+  var close = document.querySelector('.popup.is-opened .btn-close');
+  console.log(close);
+  close.focus();
```

Comment:
Hi 👋 
I'll try and help - I have made assumptions here..... unless you have a custom function called focus() that overrides jQuery's function, the variable close will need to be a jQuery object to use focus() - in its current state (using the document selector) `close` is a DOM element. This will give you an error. It would be best to do `$(close).focus()` to prevent the error. In fact you could just write `$('.popup.is-opened .btn-close').focus()`.

Reply:
This wasn't the issue - it was to do with matching things up (there are a lot of functions and classes going around in this file). Draft PRs don't need reviews - I just put bigger PRs up drafts so that they're stored somewhere other than my computer (just on the off chance anything happens to it). I then take the draft status off when they are ready for review :)

Reply:
Thanks for responding :) 
It wasn't a review Bev. Just a simple helping hand. Was it OK for me to do that?